packages/@react-spectrum/s2/chromatic/Tabs.stories.tsx

```diff
@@ -27,66 +28,72 @@ const meta: Meta<typeof Tabs> = {
 
-export const Example = (args: any) => (
-  <Tabs {...args} styles={style({width: 450, height: 256})}>
-    <TabList aria-label="History of Ancient Rome">
-      <Tab id="FoR"><Edit />Founding of Rome</Tab>
-      <Tab id="MaR">Monarchy and Republic</Tab>
-      <Tab id="Emp">Empire</Tab>
-    </TabList>
-    <TabPanel id="FoR" UNSAFE_style={{display: 'flex'}}>
-      <div style={{overflow: 'auto'}}>
-        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum non rutrum augue, a dictum est. Sed ultricies vel orci in blandit. Morbi sed tempor leo. Phasellus et sollicitudin nunc, a volutpat est. In volutpat molestie velit, nec rhoncus felis vulputate porttitor. In efficitur nibh tortor, maximus imperdiet libero sollicitudin sed. Pellentesque dictum, quam id scelerisque rutrum, lorem augue suscipit est, nec ultricies ligula lorem id dui. Cras lacus tortor, fringilla nec ligula quis, semper imperdiet ex.</p>
-      </div>
-    </TabPanel>
-    <TabPanel id="MaR" UNSAFE_style={{display: 'flex'}}>
-      <div style={{overflow: 'auto'}}>
-        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed ut vulputate justo. Suspendisse potenti. Nunc id fringilla leo, at luctus quam. Maecenas et ipsum nisi. Curabitur in porta purus, a pretium est. Fusce eu urna diam. Sed nunc neque, consectetur ut purus nec, consequat elementum libero. Sed ut diam in quam maximus condimentum at non erat. Vestibulum sagittis rutrum velit, vitae suscipit arcu. Nulla ac feugiat ante, vitae laoreet ligula. Maecenas sed molestie ligula. Nulla sed fringilla ex. Nulla viverra tortor at enim condimentum egestas. Nulla sed tristique sapien. Integer ligula quam, vulputate eget mollis eu, interdum sit amet justo.</p>
-        <p>Vivamus dignissim tortor ut sapien congue tristique. Sed ac aliquet mauris. Nulla metus dui, elementum sit amet luctus eu, condimentum id elit. Praesent id nibh sed ligula congue venenatis. Pellentesque urna turpis, eleifend id pellentesque a, auctor nec neque. Vestibulum ipsum mauris, rutrum sit amet magna et, aliquet mollis tellus. Pellentesque nec ultricies nibh, at tempus massa. Phasellus dictum turpis et interdum scelerisque. Aliquam fermentum tincidunt ipsum sit amet suscipit. Fusce non dui sed diam lacinia mattis fermentum eu urna. Cras pretium id nunc in elementum. Mauris laoreet odio vitae laoreet dictum. In non justo nec nunc vehicula posuere non non ligula. Nullam eleifend scelerisque nibh, in sollicitudin tortor ullamcorper vel. Praesent sagittis risus in erat dignissim, non lacinia elit efficitur. Quisque maximus nulla vel luctus pharetra.</p>
-      </div>
-    </TabPanel>
-    <TabPanel id="Emp" UNSAFE_style={{display: 'flex'}}>
-      <div style={{overflow: 'auto'}}>
-        <p>Alea jacta est.</p>
-      </div>
-    </TabPanel>
-  </Tabs>
-);
+export const Example = {
+  render: (args: any) => (
+    <Tabs {...args} styles={style({width: 450, height: 256})} aria-label="History of Ancient Rome">
```

Comment:
opinion, I changed S2 Tabs to require an aria-label or aria-labelledby following https://github.com/adobe/react-spectrum/pull/7600#discussion_r1924610998

I needed to move it to the Tabs component so that I could propagate it to the TabList OR the TabMenu. It's also important that the element is always in the DOM and the TabList isn't when we're collapsed.

Should I push some form of this change down to RAC Tabs? If so, what? I could remove the label props, but then extending them is weird.

We currently accept both label props on both Tabs and TabList, though putting it on RAC Tabs actually won't do anything for accessibility.

Do I remove them all (label/described/etc) from the S2 TabList?

Reply:
IMO I think it is fine to have the aria-label/labelledby props at the top level for S2 Tabs even if it differs from RAC Tabs. As for whether or not to change RAC Tabs, I guess that is dependent on whether we want to make re-creating this kind of collapsable behavior easier for end users (though they can implement a context shuttling down the aria-label/labelledby to their TabList/Tabpicker relatively easily I suppose).

 
> We currently accept both label props on both Tabs and TabList, though putting it on RAC Tabs actually won't do anything for accessibility.

Not sure I follow this, looks like you omit those label props from S2 TabList? Unless you are referencing RAC Tabs but RAC Tabs doesn't take the label props in that case.

Reply:
oh hm, well at the moment providing an `aria-label` to Tabs doesn't apply it anywhere I believe because https://github.com/adobe/react-spectrum/blob/7b7b461747b9e72adeb15eb069a57e642d3de01a/packages/react-aria-components/src/Tabs.tsx#L167 doesn't have `labelable', which on a 2nd read is what I guess you were saying with `though putting it on RAC Tabs actually won't do anything for accessibility.`... 

Actually looks like a bunch of our components (ComboBox, etc) that have a outer div wrapper have this mismatch, not sure we actually want to allow the container to have those label props...

Reply:
Ah looks like they propagate those label props to the inner input/etc so might be good to do the same for RAC Tabs

Reply:
Do we remove the labelable props on the TabList then? that'd technically be breaking

Reply:
For ComboBox/TextField, it looks like the `aria-label` provided on the `Input` wins over the one placed on the ComboBox/TextField. We could follow that behavior instead of removing the labelable props from TabList for consistency